### ISSUE TITLE
InSight lander support

### ISSUE THREAD

**Issue by stirlingalgermissen** [corincerami/mars-photo-api#80]
Are there any plans to support raw images from the InSight lander?

**Comment by chrisccerami** [comment]
I think that should be doable. I'll look into it.

**Comment by Monte9** [comment]
@chrisccerami I'd be interested to help support the InSight lander data. How would I go about doing this?

**Comment by cgibsonmm** [comment]
Hey, I will look into this support. I should be able to build it out in a few days.

**Comment by hjohnson12** [comment]
Hey everyone! Any news on an update with support for raw images from the InSight lander?

**Comment by stirlingalgermissen** [comment]
Also M2020!

**Comment by cgibsonmm** [comment]
I just need to know where to find the photos for this lander. @chrisccerami if you can point me to something I can do the rest.

**Comment by hjohnson12** [comment]
@cgibsonmm I'm not 100% sure the source of where they're scraping the images from for the current API, but there is this site with the raw images for the InSight Lander directly from NASA here if this may help: 

https://mars.nasa.gov/insight/multimedia/raw-images/?order=sol+desc%2Cdate_taken+desc&per_page=50&page=0&mission=insight

**Comment by chrisccerami** [comment]
Yes, the rover scrapers pull from NASA's raw images page for each rover (now only Curiosity following the end of Opportunity's mission).
